### PR TITLE
Minor Styling Changes

### PR DESCRIPTION
**Pull request by sampoder**
I feel like in the current one, you can't really see anything and its kind of dead... this photo feels more alive IMO.

###### I made some tiny styling changes, feel free to revert.

### CODE REVIEW

**Comment by exu3** [comment]
Thanks! I like the text styling changes.

My thoughts on the new image.
```
+ it feels alive because people are watching something intently and clapping
+ visible facial expressions

- it's a little dark
- some parts are visibly blurry (and they're close to the camera)
- it's a bit close up
```

Thoughts on the old image:
```
+ the colorful decorations on the ceiling look nice
+ there are many people 
+ the colors in the picture feel bright

- people aren't _doing_ anything (I agree, it feels a little "dead")
- it's not clear what people are looking at
- it's almost the same background as the homepage hero
```

I wonder if there's a good picture of people working on projects; I think that could make a good hero image because it paints a picture of what a hackathon might look like to someone whose never been to one before.

**Comment by sampoder** [comment]
100% agree with this feedback, let me take a look!

**Comment by sampoder** [comment]
Any of these look good?

**Comment by sampoder** [comment]
(also @ https://hackclub.slack.com/archives/C016DEDUL87/p1662478628601189)

**Comment by exu3** [comment]
Sorry for the late reply! I tested some of them out in the hero image, but they feel too zoomed in or in some of them the visible parts don't feel very representative of a high school hackathon.

Were there any particular images in that set that stood out to you?

I can take a look through the album sometime to see if any photos would work well here.

**Comment by exu3** [comment]
What if it was a timelapse?

**Comment by sampoder** [comment]
I love that idea! I actually just did this in https://github.com/hackoc/site/pull/1

File: components/hackathons/landing.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Box, Button, Heading, Text, Card } from 'theme-ui'
 import Fade from 'react-reveal/Fade'
 import ScrollHint from '../scroll-hint'
 import Image from 'next/image'
-import hero from '../../public/hackathons/assemble.JPG'
+import hero from '../../public/hackathons/assemble-talent-show.jpg'
```
Comment (sampoder) [inline components/hackathons/landing.js]:
```suggestion
import hero from '../../public/hackathons/assemble.JPG'
```

**Comment by sampoder** [comment]
Closing due to git conflicts.

**Pull request closed without merging**